File: Sum square difference.cpp
#include <iostream>
using namespace std;
int main()
{
    int i,n,suofsq=0,sqofsu=0,final=0;
    cout << "Enter the range of your ";
    cin >> n;
    for(i=0;i<=n;i++)
    {
        suofsq = suofsq + i*i;
        sqofsu = sqofsu + i;
    }
    sqofsu = sqofsu * sqofsu;
    final =sqofsu - suofsq;
    cout << final;
}

File: Sum square difference.py
i=0
n=0
suofsq=0
sqofsu=0
final=0
n = int(input("Enter the range of your "))
for i in range (0,n+1):
    suofsq = suofsq + i*i
    sqofsu = sqofsu + i
sqofsu = sqofsu * sqofsu
final =sqofsu - suofsq
print(final)